### DISCUSSION TITLE
creating-source-only-nuget-packages/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#430] [Announcements]
# creating-source-only-nuget-packages/

In this post I show how you can create a NuGet package that contains source code (instead of dlls) which is then compiled into  the target project

https://andrewlock.net/creating-source-only-nuget-packages/

**Comment by andychiare** [comment]
Another use case for source-only NuGet packages is the creation of project templates

**Comment by kjkrum** [comment]
I noticed that if you don't hide the .cs files they're editable in the consuming project, and you're actually editing the cache in `~/.nuget/packages/{packagename}`, so other consumers could be affected by your changes. This is with Rider on Linux. Is this also the case with VS on Windows?
